src/Telnyx.net/Services/Wireless/SimCards/SimCardDisableService.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Telnyx.net.Services.Wireless.SimCards
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Telnyx.net.Entities.Wireless.SimCards;
+
+    /// <summary>
+    /// SimCardsDisableService
+    /// </summary>
+    public class SimCardDisableService : Service<SimCardRecord>,
```

Comment:
Make this class internal if not intended to be exposed

Reply:
done